ui/app/rif/constants/index.js

```diff
@@ -1,26 +1,9 @@
-// import icons for each token
-import { faBitcoin, faEthereum } from '@fortawesome/free-brands-svg-icons'
-import { faCoins, faCheckCircle, faBolt, faArchive } from '@fortawesome/free-solid-svg-icons'
-/**
- * Add the icon to the array with this structure
- *   name: icon,
-*/
-export const cryptos = {
-    bitcoin: {
-        name: 'Bitcoin',
-        color: '#FFA500',
-        icon: faBitcoin,
-    },
-    ethereum: {
-        name: 'Ethereum',
-        color: '#065535',
-        icon: faEthereum,
-    },
-    rsk: {
-        name: 'RSK',
-        color: '#003366',
-        icon: faCoins,
-    },
+import { faCoins, faCheckCircle, faBolt, faArchive } from '@fortawesome/free-solid-svg-icons';
+import rifConfig from './../../../../rif.config';
```

Comment:
I recommend you to put constants inside objects to diferentiate what constants are from where, is just to organize better the constants inside this file.

Reply:
This was moved in next commit, so as you said, we have sepparated logic here.